test/test.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const db = require('../db/index');
+
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let app = require('../server/index');
+
+let should = chai.should();
+chai.use(chaiHttp);
+
+describe('/GET users', () => {
+  it('it should GET all the users', (done) => {
+    chai.request(app)
+        .get('/users')
+        .end((err, res) => {
+              res.should.have.status(200);
+              res.body.data.should.be.a('array');
+              res.body.data.length.should.equal(10); //gt is greater than
```

Comment:
Brittle. What if you decide on an eleventh user?

just check that it's greater than 1